docs_user/modules/proc_adopting-image-service-with-nfs-ganesha-backend.adoc

```diff
@@ -153,7 +149,7 @@ Mounts:
 * Check the mountpoint:
```

Comment:
“Check the mountpoint” : Can we elaborate on what customers should be looking for? For example, “Check that the mountpoint is pointing to the correct location/directory/something else?

Reply:
Check with the `mount` command that the mountpoint that points to /var/lib/glance/images  is mapped to the expected `<nfs server ip> : <nfs path>`

Reply:
How about : "Check that the mountpoint that points to `/var/lib/glance/images` is mapped to the expected `nfs server ip` and `nfs path` that you defined in the new default GlanceAPI instance?

Reply:
Sounds good to me!